test/index.js

```diff
@@ -462,7 +462,7 @@ describe('send(ctx, file)', function(){
         var app = koa();
 
         app.use(function *(){
-          yield send(this, 'test/fixtures/hello', {extensions: {}});
+          yield send(this, 'test/fixtures/hello', { extensions: {}});
```

Comment:
I'm adding refactor of all style issues I found in project but still wondering, why not use some kind of auto-formatter?